application/forms/ApiTransportForm.php

```diff
@@ -49,8 +49,7 @@ protected function assemble()
             'description'   => t('User to authenticate with using HTTP Basic Auth')
         ]);
 
-        // TODO: Use a password element
-        $this->addElement('text', 'password', [
+        $this->addElement('password', 'password', [
             'required'      => true,
             'label'         => t('API Password')
```

Comment:
We should probably add `autocomplete=new-password` here. (https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#preventing_autofilling_with_autocompletenew-password)